src/views/Media/News/NewsletterSignup.vue

```diff
@@ -0,0 +1,302 @@
+                  <h2 class="cve-top-margin">
+                    <p id="successIcon" class="is-hidden">success</p>
+                    <font-awesome-icon style="flex: 0 0 40px;" size="1x"  icon="check-circle" role="danger"
+                      aria-labelledby="successIcon" aria-hidden="false" />
+                    Service is currently unavailable.
+                  </h2>
+                  <p>Please
+                    <span class="icon-text">
+                      <a href="https://cveform.mitre.org/" target="_blank">report the issue
+                        <span class="icon is-size-7 cve-icon-xxs">
+                          <p id="extenalLink1" class="is-hidden">external site</p>
+                          <font-awesome-icon icon="external-link-alt" aria-labelledby="extenalLink1" aria-hidden="false" focusable="false"/>
+                        </span>
+                      </a>
+                    </span>
+                    and try again later. Sorry for the inconvenience.
+                  </p>
+                </div>
+              </div>
+            </div>
+            <div class="mt-2">View our <router-link to=/Legal/PrivacyPolicy>Privacy Policy</router-link></div>
+          </div>
+          <h2 class="title">How to Unsubscribe</h2>
+          <p>
+            <a href="mailto:lms@mitre.org?Subject=signoff%20cve-announce-list">Click here</a> to send email to
+            <span class="has-text-weight-bold">lms@mitre.org</span> with
+              the subject: "<span class="has-text-weight-bold">signoff cve-announce-list</span>". Alternatively, instructions on how to unsubscribe
+              appear at the bottom of each newsletter.
+          </p>
+        </main>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  name: 'NewsletterSignup',
+  components: {
+  },
+  data() {
+    return {
+      errors: [],
+      isLoading: false,
+      submitted: false,
+      subscribed: false,
+      firstRandomNum: undefined,
+      secondRandomNum: undefined,
+      mathAnswer: undefined,
+      userInfo: {
+        emailAddress: null,
+        mathAnswer: null,
+        firstName: '',
+        lastName: '',
+        jobFunction: '',
+        orgType: '',
+        organization: '',
+        cityTown: '',
+        stateProvince: '',
+        country: '',
+      },
+    };
+  },
+  methods: {
+    generateMathQuestion() {
+      this.firstRandomNum = Math.floor(Math.random() * 9);
+      this.secondRandomNum = Math.floor(Math.random() * 9);
+
+      this.mathAnswer = this.firstRandomNum + this.secondRandomNum;
+    },
+    checkForm() {
+      this.errors = [];
+      if (!this.userInfo.emailAddress) {
+        this.errors.push('Email required.');
+      } else if (!this.validEmail(this.userInfo.emailAddress)) {
+        this.errors.push('Valid email required.');
+      }
+
+      if (!this.userInfo.mathAnswer) {
+        this.errors.push('Response to 9 + 4 required');
```

Comment:
This should be generic, like "Response to Math question required" or just "Please complete required fields"